src/frontend/src/lib/services/request-pouh-credential.services.ts

```diff
@@ -23,17 +24,16 @@ const handleSuccess = async (
 	const { auth: authI18n } = get(i18n);
 	if ('Ok' in response) {
 		// TODO: GIX-2646 Add credential to backend and load user profile
-		const fakeTemporaryCredentialSummary = {
-			credential_type: POUH_CREDENTIAL_TYPE,
-			verified_date_timestamp: BigInt(Date.now()),
-			expire_date_timestamp: BigInt(Date.now() + 1000 * 60 * 60 * 24 * 365)
+		const fakeTemporaryCredentialSummary: UserCredential = {
+			credential_type: { [POUH_CREDENTIAL_TYPE]: null },
+			verified_date_timestamp: [BigInt(Date.now())],
```

Comment:
`toNullish()`?

(for this line an other similar lines)

Reply:
this will be removed anyway.